spec/features/matomo_spec.rb

```diff
@@ -3,9 +3,15 @@
 require 'rails_helper'
 
 RSpec.describe 'Matomo Traking Code' do
-  before { Capybara.ignore_hidden_elements = false }
+  before do
+    ENV['MATOMO_TEST'] = 'true'
+    Capybara.ignore_hidden_elements = false
+  end
 
-  after { Capybara.ignore_hidden_elements = true }
+  after do
+    Capybara.ignore_hidden_elements = true
+    ENV.reject! { |k, _v| k == 'MATOMO_TEST' }
```

Comment:
Since this is a test, I don't think it matters -- but I wonder if something like `ENV.delete('MATOMO_TEST')` would be more sturdy

Reply:
Ah yeah that's cleaner.  I'll change it.